tmva/tmva/inc/LinkDef1.h

```diff
@@ -68,8 +70,9 @@
 #pragma link C++ class TMVA::MethodCrossValidation+;
 #pragma link C++ class TMVA::MethodDL+;
 
+#ifdef R__HAS_DATAFRAME
 // BDT inference
 #pragma link C++ class TMVA::Experimental::RBDT<TMVA::Experimental::BranchlessForest<float>>;
```

Comment:
It looks like this is a bit too restrictive, and we are creating more depedencies than necessary. As a hotfix, it's probably ok, but can this also work without dataframe?

Reply:
In principle yes, however, TMVA experimental has RDF as dependency and therefore we carry this even to some classes, which are actually not dependent on the RDF code itself. The fine granular splitting of the dependencies has a significant overhead and therefore I would go for this solution.